test/ts/unit/collateralized_simple_interest_terms_contract.ts

```diff
@@ -665,10 +665,12 @@ contract("CollateralizedSimpleInterestTermsContract (Unit Tests)", async (ACCOUN
 
             before(async () => {
                 const INTEREST_RATE_SCALING_FACTOR = await termsContract.INTEREST_RATE_SCALING_FACTOR.callAsync();
+                const PERCENTAGE_SCALING_FACTOR = await termsContract.PERCENTAGE_SCALING_FACTOR.callAsync();
 
                 const TOTAL_INTEREST = principalAmount
                     .mul(interestRate)
-                    .div(INTEREST_RATE_SCALING_FACTOR);
+                    .div(INTEREST_RATE_SCALING_FACTOR)
```

Comment:
So the fact that our tests mimic our actual code seems to be a problem. Thinking that we might want to have rawer values in our tests to ensure that the production code is generating what we expect. This could also serve as a form of documentation.

Reply:
Totally agree -- fixing.